### PR TITLE
AppConfig/advertiseSubscriptions as a safety switch for SDK

### PR DESCRIPTION
**Pull request by dorjesinpo**
Control enabling configureStream in new SDK release

### CODE REVIEW

File: src/groups/mqb/mqba/mqba_sessionnegotiator.cpp [hunk 1]
```diff
@@ -146,6 +146,17 @@ void loadBrokerIdentity(bmqp_ctrlmsg::ClientIdentity* identity,
             .append(bmqp::MessagePropertiesFeatures::k_FIELD_NAME)
             .append(":")
             .append(bmqp::MessagePropertiesFeatures::k_MESSAGE_PROPERTIES_EX);
+
+        const mqbcfg::AppConfig& theConfig = mqbcfg::BrokerConfig::get();
+
+        if (theConfig.brokerVersion() == 999999 ||
+            (theConfig.configureStream() &&
+             theConfig.advertiseSubscriptions())) {
+            features.append(";")
```
Comment (kaikulimu) [inline src/groups/mqb/mqba/mqba_sessionnegotiator.cpp:155]: Is the trigger to enable subscription to append this "SUBSCRIPTIONS:CONFIGURE_STREAM"?

Comment (dorjesinpo) [reply src/groups/mqb/mqba/mqba_sessionnegotiator.cpp:155]: Yes!

Comment (dorjesinpo) [reply src/groups/mqb/mqba/mqba_sessionnegotiator.cpp:155]: We need to add this to https://github.com/bloomberg/blazingmq/pull/224

**Review by kaikulimu — APPROVED** [review]
lgtm

**Pull request merged**